portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -65,15 +68,17 @@
     String content;
     String imageUrl;
     String mood;
+    float sentiment;
     String cursor;
 
-    private Comment(long id, String name, Long timestamp, String content, String imageUrl, String mood, String cursor) {
+    private Comment(long id, String name, Long timestamp, String content, String imageUrl, String mood, float sentiment, String cursor) {
```

Comment:
Nit: try to adhere to having less than [100](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit) characters per line.

This is much more important in a codebase with many people contributing / reading, and if you have a sufficiently large monitor it's really up to you. But right now I can't see all of it in my code review and have to scroll sideways which hurts readability.